docs/hardis/misc/custom-label-translations.md

```diff
@@ -0,0 +1,153 @@
+# hardis:misc:custom-label-translations
+
+## Description
```

Comment:
@matheus-delazeri the doc page is automatically generated from property "description" in the command, please move your text there or it will be overwritten ^^